apps/extension/src/ui/apps/dashboard/routes/Portfolio/AccountContextMenu.tsx

```diff
@@ -115,35 +132,38 @@ export const AccountContextMenu = forwardRef<HTMLElement, Props>(function Accoun
         {trigger ? trigger : <MoreHorizontalIcon className="shrink-0" />}
       </ContextMenuTrigger>
       <ContextMenuContent className="border-grey-800 z-50 flex w-min flex-col whitespace-nowrap rounded-sm border bg-black px-2 py-3 text-left text-sm shadow-lg">
-        {account && (
-          <>
-            {canToggleIsPortfolio && (
-              <ContextMenuItem onClick={toggleIsPortfolio}>{toggleLabel}</ContextMenuItem>
-            )}
-            {canCopyAddress && (
-              <ContextMenuItem onClick={copyAddress}>{t("Copy address")}</ContextMenuItem>
-            )}
-            {canRename && (
-              <ContextMenuItem onClick={openAccountRenameModal}>{t("Rename")}</ContextMenuItem>
-            )}
-            {canExport && (
-              <ContextMenuItem onClick={openAccountExportModal}>
-                {t("Export as JSON")}
-              </ContextMenuItem>
-            )}
-            {canExportPk && (
-              <ContextMenuItem onClick={openAccountExportPkModal}>
-                {t("Export private key")}
+        <Suspense>
+          {account && (
+            <>
+              {canToggleIsPortfolio && (
+                <ContextMenuItem onClick={toggleIsPortfolio}>{toggleLabel}</ContextMenuItem>
+              )}
+              {canCopyAddress && (
+                <ContextMenuItem onClick={copyAddress}>{t("Copy address")}</ContextMenuItem>
+              )}
+              {account && <ViewOnExplorerMenuItem account={account} />}
```

Comment:
Don't need a check for `account` here, it's already inside one.
```suggestion
              <ViewOnExplorerMenuItem account={account} />
```